lib/allowed_ops.ml

```diff
@@ -79,15 +100,21 @@ module Make (B : BEHAVIOUR) : S = struct
                 Lwt.return_unit
             | e -> Lwt.fail e)
     in
+    (if skip_serial then Lwt.return_unit
+    else
```

Comment:
Indentation seems confusing to me - even if ocp-indent suggests this. The conditional code is not indented.